WordPress/wordpress-playground
# CI: Check if both CJS and ESM builds succeeded

Adds the `test:esmcjs` check to GitHub CI, as it was missing.

Many Playground modules are built both to ES Modules for the browser and to CommonJS modules for Node.js. The `test:esmcjs` check runs after the build process and tries to both `import` and `require` the built modules in Node.js to confirm both forms can be used.

This PR also fixes a bug in that check to ensure both the `import` and `require` test scripts actually run.

## Testing instructions

Confirm the CI build job is green and confirm the run included the `test:esmcjs` check.

Extracted from https://github.com/WordPress/wordpress-playground/pull/851.
